src/main/java/org/emela/chautari/controller/LoginApiController.java

```diff
@@ -0,0 +1,26 @@
+package org.emela.chautari.controller;
+
+import org.emela.chautari.api.LoginApi;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.context.request.NativeWebRequest;
+import java.util.Optional;
+@javax.annotation.Generated(value = "org.openapitools.codegen.languages.SpringCodegen", date = "2020-04-18T09:53:05.107-04:00[America/New_York]")
+
+@Controller
+@RequestMapping("${openapi.chautariRentalService.base-path:/chautari}")
```

Comment:
lets not auto generate the actual implementations.

Reply:
These controllers are just a sample placeholder. This will be replaced by the implemented controller classes.